task2/src/app.ts

```diff
@@ -2,6 +2,28 @@ import * as express from 'express';
 
 import userRouter from './routes/index';
 
+import seq from './schema/index';
+import UserModel from './schema/user';
+
+seq
+  .authenticate()
+  .then(() => {
+    console.log('Connection has been established successfully.');
+  })
+  .catch(err => {
+    console.error('Unable to connect to the database:', err);
+  });
+
+// UserModel.sync({ force: true }).then(() => {
+//   UserModel.create({
+//     id: 'qwe',
+//     login: 'qwe',
+//     password: 'qwe',
+//     age: 27,
+//     isDeselected: false
+//   });
+// });
+
 const app = express();
```

Comment:
Move a server setup to separate folder/file;

Reply:
done, thanks